crates/matrix-sdk/src/oidc.rs

```diff
@@ -350,7 +348,8 @@ impl Oidc {
         OidcLoginBuilder::new(self.clone())
     }
 
-    /// Authorize the given scope by using the Authorization Code flow.
+    /// Get the URL that should be presented to authorize a given scope with the
```

Comment:
Sorry I forgot to add a comment earlier, we need to add a sentence in those docs to call the finish method when the user is redirected, since this method is useless without the other.